README.md

```diff
@@ -233,4 +233,16 @@ Your app is ready to be deployed! Note that builds for deployment are not made o
     where u.email = @email
     ```
 
-14. Return to the UI and you should be good to go!
+14. Grant participant permission to your account
+
+    - Open the Keycloak Admin Console (http://localhost:18080/admin/) and log in as an administrator (see above).
```

Comment:
nit: maybe an ordered list rather than unordered would be easier to follow?

Reply:
I changed this around after realizing most of this was already listed in a Keycloak specific guide.